crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -3718,4 +3733,27 @@ mod tests {
             ]
         );
     }
+
+    #[async_test]
+    async fn test_room_member_updates_sender_and_receiver() {
+        use assert_matches::assert_matches;
+
+        let client = logged_in_base_client(None).await;
+        let room = client.get_or_create_room(room_id!("!a:b.c"), RoomState::Joined);
+
+        let mut receiver = room.room_member_updates_sender.subscribe();
+
+        assert!(receiver.is_empty());
+
+        room.room_member_updates_sender
+            .send(RoomMembersUpdate::FullReload)
+            .expect("broadcasting a room members update failed");
```

Comment:
I don't think we should send something manually; instead, we should test that the described behavior (will be pinged upon /members or incremental room member update) is correct. So we'd need two tests:

- when I'm calling /members, then I get a `FullReload`
- when I'm receiving new member info from sync, then I get `Partial` with the correct user id set

Does it make sense?